lib/pos_blockchain_test.go

```diff
@@ -1891,8 +1870,6 @@ func testProcessBlockPoS(t *testing.T, testMeta *TestMeta) {
 		var malformedOrphanBlock *MsgDeSoBlock
 		malformedOrphanBlock = _generateRealBlock(testMeta, 18, 18, 9273, testMeta.chain.BlockTip().Hash, false)
 		malformedOrphanBlock.Header.PrevBlockHash = randomHash
-		// Modify anything to make the block malformed, but make sure a hash can still be generated.
-		malformedOrphanBlock.Header.TxnConnectStatusByIndexHash = randomHash
```

Comment:
We still need to modify something to make sure the block is malformed such that the test passes.